README.md

```diff
@@ -22,7 +22,7 @@ tactical domain-driven design patterns. The use of CQRS (separation between writ
 
 Because of this, the following assumptions are made:
  - **Repositories** are being used to persist and retrieve (by ID) **aggregates**
- - **Aggregates** are composed of one or more **entities**, and may also contain **value objects**
+ - **Aggregates** are a cluster of domain objects that can be treated as a single unit. They are usually composed of one or more **entities**, and may also contain **value objects**
```

Comment:
Yes, if an aggregate consists of only one entity, then that entity is automatically the Aggregate Root.